src/Microsoft.ML.CodeGen/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -41,7 +41,7 @@ public void GenerateOutput()
             SetRequiredNugetPackages(trainerNodes, ref includeLightGbmPackage, ref includeMklComponentsPackage, ref includeFastTreeePackage);
 
             // Get Namespace
-            var namespaceValue = Utils.Normalize(_settings.OutputName);
+            var namespaceValue = _settings.OutputName;
```

Comment:
>_settings.OutputName [](start = 33, length = 20)

Shouldn't this be `Sanitize(_settings.OutputName)`?

Reply:
The `OutputName` is sanitized here https://github.com/dotnet/machinelearning/pull/4212/commits/672a689c7e6c3fdaa2e39ce939edd311455da325#diff-8c190379bfcd845d57ebdddf120d1c96R354 because it's used in many places